src/main/scala/com/github/saurfang/sas/spark/DefaultSource.scala

```diff
@@ -47,6 +47,24 @@ class DefaultSource
                                sqlContext: SQLContext,
                                parameters: Map[String, String],
                                schema: StructType): SasRelation = {
-    SasRelation(checkPath(parameters), schema)(sqlContext)
+    val path = checkPath(parameters)
+    
+    val forceLowerCaseColumnNames = parameters.getOrElse("forceLowerCaseColumnNames", "false")
```

Comment:
minor: should there be an option `forceUpperCaseColumnNames` too?

Reply:
I dont think we need this, as is pretty bad spark practice?